.github/workflows/neon_workflow.yml

```
@@ -0,0 +1,223 @@
+          # Health check on production
+          curl -f https://skyyrose.co/api/v1/consensus/health || exit 1
+          curl -f https://skyyrose.co/api/v1/content/health || exit 1
+          curl -f https://skyyrose.co/api/v1/ecommerce/health || exit 1
+
+      - name: Notify deployment success
+        if: success()
+        uses: 8398a7/action-slack@v3
+        with:
+          status: ${{ job.status }}
+          text: '🚀 DevSkyy deployed to production successfully!'
+          webhook_url: ${{ secrets.SLACK_WEBHOOK }}
```

actionlint (1.7.8)

input "webhook_url" is not defined in action "8398a7/action-slack@v3". available inputs are "author_name", "cancelled_message", "channel", "custom_payload", "failure_message", "fields", "github_base_url", "github_token", "icon_emoji", "icon_url", "if_mention", "job_name", "mention", "status", "success_message", "text", "username"

(action)